python-checks/src/main/java/org/sonar/python/checks/UselessStatementCheck.java

```diff
@@ -122,13 +120,13 @@ private static void checkNode(SubscriptionContext ctx) {
     if (parent == null || !parent.is(Kind.EXPRESSION_STMT)) {
       return;
     }
-    if (isWithinContextlibSuppress(tree)) {
+    if (isWithinSuppressionContext(tree)) {
```

Comment:
Super minor thing but we tend to use `ignored` (like `isWithinIgnoredContext`) or possibly `isException` (though here it may be confusing with actual exceptions) for silenced issues - to have consistent terminology.

The prior `suppress` in the method name was related to the fact we were targeting the `contextlib.suppress` context manager only. Keeping the same name to refer to suppressed issues may make things confusing.